assets/js/googlesitekit/datastore/user/key-metrics.test.js

```diff
@@ -415,10 +453,7 @@ describe( 'core/user key metrics', () => {
 
 					expect(
 						registry.select( CORE_USER ).getAnswerBasedMetrics()
-					).toEqual( [
-						...expectedMetrics,
-						...conversionTailoredMetrics,
-					] );
+					).toEqual( conversionTailoredMetrics );
 
 					// Conversion Tailored Metrics should be added to the list if the
 					// includeConversionTailoredMetrics setting is true.
```

Comment:
It would be good to update this comment to be a bit more accurate, e.g. something like this:

```suggestion
					// Conversion Tailored Metrics should be included in the list if the
					// includeConversionTailoredMetrics setting is true.
```